packages/primitives/src/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from '@emotion/styled'
+import { Theme, css } from 'theme-ui'
+import { SerializedStyles } from '@emotion/serialize'
+import {
+  StyledSystemProps,
+  variant,
+  sx,
+  shouldForwardProp,
+  styledSystemProps,
+} from '../Box/Box'
+
+/**
+ * base
+ *
+ * Generate the base CSS for the button component
+ * that is aware of the Theme UI theme
+ *
+ * @param props
+ */
+export const baseStyles = ({ theme }: { theme: Theme }): SerializedStyles =>
+  css({
+    display: 'block',
+    width: '100%',
+    p: 2,
+    appearance: 'none',
+    fontSize: 'inherit',
+    lineHeight: 'inherit',
+    border: '1px solid',
```

Comment:
should we start with borders as `grey.3` or add a color.border to the theme?

Reply:
Yea, I would add a default border color so that we don' thave to always use `grey.3`

Reply:
Merging for now but we can add it when we use borders again later.